### PR TITLE
B lorient618

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient618.xml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+<msItem xml:id="ms_i2.1">
+<locus from="54r"/>
+<title type="complete" ref="LIT1297Dersan#HomilyChrysostom"/>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.2">
+<locus from="71v"/>
+<title type="complete" ref="LIT1297Dersan#Miracles"/>
+<note>The history of Ṭobit on <locus target="#76v">f. 76v</locus>.</note>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient618.xml:90]: I would encode it as a subunit: ms_i2.2.1

File: LondonBritishLibrary/orient/BLorient618.xml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        <q xml:lang="gez">ድርሳነ፡ ገብርኤል፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="e1">
+<locus target="#53v"/>
+<desc>The name of the scribe is <persName role="scribe" ref="PRS13527Fiqtor"><foreign xml:lang="gez">ፊቅጦር፡</foreign></persName>.</desc>
+</item>
+
+<item xml:id="e2">
+<locus target="98"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient618.xml:143]: #98

File: LondonBritishLibrary/orient/BLorient618.xml [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+<desc>Writing on <locus target="#53v">f. 98</locus> erased.</desc>
+</item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient618.xml:145]: 53v or 98?

File: LondonBritishLibrary/orient/BLorient618.xml [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+<item xml:id="e3">
+<locus target="#1r #97r"/>
+<desc>Name of the first owner erased, and that of <persName role="owner" ref="PRS13518KeflaGiyorgis"><foreign xml:lang="gez">ክፍለ፡ ጊዮርጊስ፡</foreign></persName> substituted.</desc>
+</item>
+
+<item xml:id="e4">
+<desc>Later owners: <persName role="owner" ref="PRS13519MalkeaIyasus"><foreign xml:lang="gez">መልክአ፡ ኢየሱስ፡</foreign></persName>,
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient618.xml:153]: In another commit PRS13519MalkeaIyasus is encoded as an owner of BLorient 611. This commit is already merged. So please, if there is some confusion creat a new branch for corrections. The same is at least for the two other.

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient618.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient618.xml:143]: Sure

File: LondonBritishLibrary/orient/BLorient618.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient618.xml:145]: 98

File: LondonBritishLibrary/orient/BLorient618.xml (hunk 4 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient618.xml:153]: I assumed that four names which appear both in BLorient611 and BLorient 618 are the same people, it's too much of a coincidence imo.

Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient618.xml:220]: Thank you! Would you mind adding the relation to this manuscript in their files as well?

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**